src/assets/content/dictionary/application-programming-interface-api.md

```diff
@@ -1,6 +1,10 @@
----
-title: Application Programming Interface (API)
-definition: ''
-perspectives: []
+
+title: Cascading Style Sheets (CSS)
+definition: CSS stands for Cascading Style Sheets and is used to stylize elements written in a markup language such as HTML. It separates the content from the visual representation of the site.
+sources:
+- sourceurl: https://www.hostinger.com/tutorials/what-is-css
+perspectives:
+- meaning: with CSS, we are able to create rules, and apply those rules to many elements within the website. This approach offers many advantages when site-wide changes are required by a client
```

Comment:
```suggestion
- meaning: a way to create rules, and apply those rules to many elements within the website. This approach offers many advantages when site-wide changes are required by a client
```
suggestion: Since our code concats the role and meaning together, I've made a suggestion to write the meaning in the style of `As a <role>, "<term>" means <meaning>.` You're welcome to accept the suggestion using the Github UI, or make another commit to correct the perspective.